packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,120 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[$style['c-mediaElement'], { [$style['c-mediaElement--stack']]: stacked }, { [$style['c-mediaElement--reverse']]: reverse }]">
+        <div :class="[$style['c-mediaElement-content'], contentAlignClass]">
+            <h3 :class="[$style['c-mediaElement-title'], $style['c-mediaElement-text'], textSizeClass]">
+                {{ title }}
+            </h3>
+            <p :class="[$style['c-mediaElement-text'], textSizeClass]">
+                {{ text }}
+            </p>
+        </div>
+        <div :class="[$style['c-mediaElement-imgWrapper'], imageAlignClass]">
+            <img
+                :class="$style['c-mediaBlock-img']"
```

Comment:
```suggestion
                :class="$style['c-mediaElement-img']"
```

Reply:
I have modified thanks, I have also added a class to allow anyone to override the image styles, for example incase they have specific needs like a border. Until I have a greater idea of what people need style's wise on the image then this I thought for now would be the best route.